frontend/src/components/common/Resource.tsx

```diff
@@ -331,23 +332,32 @@ export function SectionGrid(props: SectionGridProps) {
 
 export function DataField(props: TextFieldProps) {
   const { label, value, ...other } = props;
+  let language = ((label as string).split('.').pop() as string);
+  if (language !== 'json') {
+    language = 'yaml';
+  }
   return (
-    <TextField
-      label={label}
-      InputProps={{
-        readOnly: true,
-      }}
-      InputLabelProps={{
-        shrink: true,
-        style: {fontSize: '1.3rem'}
-      }}
-      variant="outlined"
-      fullWidth
-      multiline
-      rowsMax="20"
-      value={value}
-      {...other}
-    />
+    <>
+      <TextField
+        label={label}
+        InputLabelProps={{
+          shrink: true,
+          style: {fontSize: '1.3rem'}
+        }}
+        variant="outlined"
+        fullWidth
+        multiline
+        rowsMax="20"
+        {...other}
+      />
+      <Editor
+        value={value as string}
+        height="40vh"
+        theme="vs-dark"
```

Comment:
The Editor shoud also have some changes in order for it to look a bit better. The theme in this case should be white, and its height should be dependent on its contents (let's try to use a 1/3 of the current default height if the lines are < 10). Unless it can be made automatically (adapt to the height of the contents, with a maximum height of what you have set up).

If possible let's try to hide the line numbers and the cursor (the main purpose of this view is to read the contents).
If a light theme for the Editor looks better we could also add it but I am not yet 100% sure on that.